### ISSUE TITLE
ES6 Features

### ISSUE THREAD

**Issue by asapzacy** [socketsupply/union-app-studio#35]
Maybe I'm missing something, but es6 spread syntax should be included by default, right? If scratches is an electron app, and assuming node/v8 is somewhat current, I should be able to use `...` without babel and/or importing packages?

```
const x = { name: 'zac', age: 99 }
const y = {
  ...x,
  location: 'world'
}
console.log(y) // SyntaxError: Unexpected token ...
```

Probably missing something basic, let me know!

**Comment by blogscot** [comment]
Someone correct me if I'm mistaken, but **scratches** depends on **electron** which is currently built on [Node v8.2.1](https://electronjs.org/); according to the Node [docs](https://node.green/#ES2018-features-object-rest-spread-properties) the object rest spread properties are available as default in version 8.3 and above.

However, I notice now that Node version 8.2.1 does support object rest properties via a harmony flag. I don't know if electron is configurable to support this.

# Update

Please see pull request #36

**Comment by heapwolf** [comment]
This should be addressed by #36.
